compiler/aarch64/runtime/ARM64arrayCopy.spp

```diff
@@ -38,76 +38,30 @@
 	.text
 	.align	2
 
-// These are entry points for halfword, word and doubleword alignment of arrays to be copied
-// in forward direction to avoid multiple checks in generic forward copy entry point,
-// when the alignment is known.
-//
-// in:    x0 - length in bytes
-//        x1 - src addr
-//        x2 - dst addr
-
+// Obsolete entry points to be removed
```

Comment:
What is preventing you from removing them now?

Reply:
There are dependencies in the OpenJ9 side.
I am keeping these entry points for now to avoid coordinated merge.

Reply:
I opened a draft PR https://github.com/eclipse-openj9/openj9/pull/19470.
I am going to remove these entry points from ARM64arrayCopy.spp after the PR above is merged,

Reply:
I opened a draft PR #7338 for removing the obsolete entry points.